File: Assets/AltTester/AltDriver/Commands/InputActions/AltKeyDown.cs
using System;
using System.Threading;

namespace Altom.AltDriver.Commands
{
    public class AltKeyDown : AltBaseCommand
    {
        AltKeyDownParams cmdParams;

        public AltKeyDown(IDriverCommunication commHandler, AltKeyCode keyCode, float power) : base(commHandler)
        {
            this.cmdParams = new AltKeyDownParams(keyCode, power);
        }
        public void Execute()
        {
            CommHandler.Send(cmdParams);
            var data = CommHandler.Recvall<string>(cmdParams);
            Thread.Sleep(100);
            ValidateResponse("Ok", data);
        }
    }
}

File: Bindings~/java/src/main/java/ro/altom/alttester/Commands/InputActions/AltKeyDown.java
package ro.altom.alttester.Commands.InputActions;

import ro.altom.alttester.IMessageHandler;
import ro.altom.alttester.Commands.AltBaseCommand;

public class AltKeyDown extends AltBaseCommand {

    private AltKeyDownParams altKeyDownParameters;

    public AltKeyDown(IMessageHandler messageHandler, AltKeyDownParams altKeyDownParameters) {
        super(messageHandler);
        this.altKeyDownParameters = altKeyDownParameters;
    }

    public void Execute() throws InterruptedException {
        SendCommand(altKeyDownParameters);
        recvall(altKeyDownParameters, String.class);
        Thread.sleep(100);
    }
}
